d2l-input-radio-styles.html

```diff
@@ -2,13 +2,6 @@
 <link rel="import" href="../d2l-colors/d2l-colors.html">
 <link rel="import" href="../d2l-typography/d2l-typography-shared-styles.html">
 
-<!--
```

Comment:
Strangely, the modularizer was placing this block at the bottom of the output file, and appending a semi-colon that would cause linting to fail.  Trying to remove (which is consistent with other style module).